Exception Handling/Task1/bin/Debug/netcoreapp3.1/Task1.deps.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@IhorStetskyi, for the following tasks, please add default .gitignore for the .net project. That help clearly reviews your code.

Reply:
gitignore added